src/ml/neural_net/tf_compute_context.cpp

```diff
@@ -201,18 +201,28 @@ class tf_image_augmenter : public float_array_image_augmenter {
 
 tf_image_augmenter::tf_image_augmenter(const options& opts) : float_array_image_augmenter(opts) {}
 
-float_array_image_augmenter::float_array_result tf_image_augmenter::prepare_augmented_images(
+float_array_image_augmenter::float_array_result
+tf_image_augmenter::prepare_augmented_images(
     float_array_image_augmenter::labeled_float_image data_to_augment) {
+  opts_ = tf_image_augmenter::get_options();
   float_array_image_augmenter::float_array_result image_annotations;
 
   call_pybind_function([&]() {
     // Import the module from python that does data augmentation
     pybind11::module tf_aug = pybind11::module::import(
         "turicreate.toolkits.object_detector._tf_image_augmenter");
 
+    const size_t output_height = opts_.output_height;
+    const size_t output_width = opts_.output_width;
+    bool resize_only = false;
+    if (opts_.crop_prob == 0.f) {
```

Comment:
Worth a TODO that in theory you need to test all the augmentation probabilities, but in practice this suffices until we plumb through all the options to Python

Reply:
Okay